src/components/css/addBooks.css

```diff
@@ -0,0 +1,58 @@
+.FormContianer-all {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.FormContianer {
+  border-top: 1px solid white;
+  display: flex;
+  flex-direction: column;
+  height: 10.625rem;
+  margin: 1.938rem 0.063rem 2.5rem 0;
+  border-radius: 4px;
+}
+
+.title {
+  display: flex;
+  text-align: left;
+  color: #666;
+  font-weight: bold;
+}
+
+.form {
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+}
+
+.input-container {
+  display: flex;
+}
```

Comment:
- This button is not styled as it is in the design template, please fix this 🙏 
Your project:
![rwgawrga](https://github.com/AliShahSafdari/bookstore/assets/73607512/b8a4df1d-4874-493d-9be6-79bdea543576)

Design template:
![ehehs](https://github.com/AliShahSafdari/bookstore/assets/73607512/28ecf6d0-9982-44cf-9897-0891180009b2)

Reply:
Thanks, I make it.